types/chrome/index.d.ts

```diff
@@ -7716,7 +7716,7 @@ declare namespace chrome {
             /** The print job title. */
             title: string;
             /** Print ticket in  CJT format. */
-            ticket: Object;
+            ticket: object;
```

Comment:
This is another thing where it's an interfaceable JSON object, but it would be a lot of time. [Here's](https://developers.google.com/cloud-print/docs/cdd#cjt) the spec of the object, referenced [here](https://developer.chrome.com/docs/extensions/reference/api/printerProvider#type-PrintJob) in the docs. I vote key-unknown.